orc8r/cloud/go/obsidian/swagger/handlers/handlers.go

```diff
@@ -14,17 +14,50 @@
 package handlers
 
 import (
+	"bytes"
+	"html/template"
 	"net/http"
+	"strings"
 
 	"magma/orc8r/cloud/go/obsidian"
 	"magma/orc8r/cloud/go/obsidian/swagger"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/lib/go/registry"
 
 	"github.com/labstack/echo"
+	"github.com/labstack/echo/middleware"
+	"github.com/pkg/errors"
 )
 
-// GetGenerateCombinedSpecHandler returns a routing handler which creates
+// RegisterSwaggerHandlers registers routes for Swagger specs and
+// the Swagger UI.
+func RegisterSwaggerHandlers(e *echo.Echo) error {
```

Comment:
This is the only identifier (function/object/type) that needs to be public in this file, the rest can be private. Once you've done that, reorder your functions such that they approximately follow a top-down call hierarchy. E.g. `registerSwaggerSpecHandlers` should be near the top, under `RegisterSwaggerHandlers`, because it's called directly by `RegisterSwaggerHandlers`